docs/installation.rst

```diff
@@ -1,13 +1,101 @@
+
+Using Metagov with a Django-based Driver
+########################################
+
+If you're using PolicyKit as your Driver, head over to the `PolicyKit Documentation <https://policykit.readthedocs.io/>`_ for instructions on how to install PolicyKit on your server. The following instructions should work for all Django-based Drivers, not just PolicyKit.
+
+Requiring Metagov
+^^^^^^^^^^^^^^^^^
+
+Install the most recent version of Metagov directly or by adding it to your project's requirements.txt:
+
+.. code-block:: shell
+
+    pip install -e git+https://github.com/metagov/gateway.git#egg=metagov&subdirectory=metagov
+
+You can then add Metagov to your project's ``settings.py``. Note that you need to install each plugin that you want to use individually:
+
+.. code-block:: python
+
+    INSTALLED_APPS = [
+        ... # existing apps
+        'metagov.plugins.slack',
+        'metagov.plugins.opencollective',
+        'metagov.plugins.github',
+        'metagov.plugins.sourcecred',
+        'metagov.plugins.loomio',
+        'metagov.plugins.discord',
+        'metagov.plugins.discourse',
+        'metagov.plugins.example', #for testing
+        'metagov.core',
+    ]
+
+You should also add a variable named METAGOV_SETTINGS to your ``settings.py``. This is used to configure plugins, for example:
+
+.. code-block:: python
+
+    METAGOV_SETTINGS = {
+        "SLACK": {
+            "CLIENT_ID": env("SLACK_CLIENT_ID", default=None),
+            "CLIENT_SECRET": env("SLACK_CLIENT_SECRET", default=None),
+            "SIGNING_SECRET": env("SLACK_SIGNING_SECRET", default=None),
+            "APP_ID": env("SLACK_APP_ID", default=None),
+        }
+    }
+
+Read more about :ref:`installing and configuring plugins<Plugins>`.
+
+Signal Handlers
+^^^^^^^^^^^^^^^
+
+When Metagov receives an event from a third-party platform, it emits a `platform_event_created` `Signal <https://docs.djangoproject.com/en/4.0/topics/signals/>`_. Your Driver should use signal receivers to capture these events and perform actions, for example:
+
+.. code-block:: python
+
+    from django.dispatch import receiver
+
+    @receiver(platform_event_created, sender=Github)
+    def github_event_receiver(sender, instance, event_type, data, initiator, **kwargs):
+        logger.debug(f"Received {event_type} event from {instance}")
+        # custom logic here
+
+Metagov also emits a `governance_process_updated` signal when a GovernanceProcess is updated. Again, use receivers to capture the signal, for example:
+
+.. code-block:: python
+
+    @receiver(governance_process_updated, sender=GithubIssueReactVote)
+    def github_vote_updated_receiver(sender, instance, status, outcome, errors, **kwargs):
+        ...
+
+Note that in both of these examples, we use the sender parameter in the receiver decorator to filter out events and governance process updates not relevant to a specific platform. Your Driver may not perform this kind of filtering - it is optional.
```

Comment:
Ah so yes, this is true, but if they want to handle all Plugin events with a single handler they can't filter it using `sender`. Instead it needs to be done like this:

```
@receiver(platform_event_created)
def metagov_event_receiver(sender, instance, event_type, data, initiator, **kwargs):
    # Need to check if this is a Plugin using subclass
    # instead of using `sender` because these are Proxy models.
    if not issubclass(sender, Plugin):
        return
    # do something...
```

Theres a linked discussion here https://stackoverflow.com/questions/22465555/django-signals-receiver-and-proxy-model I'm not sure if there's anything we can do on the metagov side to link up the signal to the parent class.